prac_06/guitar.py

```diff
@@ -0,0 +1,22 @@
+"""Guitar Class"""
+
+
+class Guitar:
+    def __init__(self, name, year=0, cost=0):
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        return "{} ({}) : ${}".format(self.name, self.year, self.cost)
+
+    def get_age(self):
+        current_year = 2021
```

Comment:
Works fine for this program, but maybe a CONSTANT would be useful for easily updating?